app/models/user.rb

```diff
@@ -0,0 +1,35 @@
+require 'bcrypt'
+class User < ActiveRecord::Base
+  has_many :recipes
+  has_many :lists
+
+  include BCrypt
+
+  def password
+    @password ||= Password.new(password_hash)
+  end
+
+  def password=(new_password)
+    @password = Password.create(new_password)
+    self.password_hash = @password
+  end
+
+  def create
+    @user = User.new(params[:user])
+    @user.password = params[:password]
+    @user.save!
+  end
+
+  def login
```

Comment:
login is not the responsibility of a model. login has to do with the web app later, so sinatra.